src/gentropy/method/l2g/trainer.py

```diff
@@ -44,10 +62,11 @@ class LocusToGeneTrainer:
     # Initialise vars
     features_list: list[str] | None = None
     label_col: str = "goldStandardSet"
-    x_train: pd.DataFrame | None = None
-    y_train: pd.Series | None = None
-    x_test: pd.DataFrame | None = None
-    y_test: pd.Series | None = None
+    x_train: np.ndarray | None = None
```

Comment:
I guess there are no categorical features ?

Reply:
We have the credible set confidence, but it is numerical